cdap-watchdog/src/main/java/co/cask/cdap/logging/appender/file/FileLogAppender.java

```diff
@@ -131,7 +132,7 @@ public void start() {
       logSchema = new LogSchema().getAvroSchema();
       FileMetaDataManager fileMetaDataManager = new FileMetaDataManager(tableUtil,
                                                                         txExecutorFactory,
-                                                                        locationFactory);
+                                                                        locationFactory, dsFramework);
```

Comment:
Yes because we changed the constructor. So it should else its a compilation error.